src/app/api/teams/route.ts

```diff
@@ -0,0 +1,20 @@
+import type { NextRequest } from 'next/server'
+
+import prisma from '@db/prismaSingleton'
+
+export const GET = async (request: NextRequest) => {
+  const sportId = request.nextUrl.searchParams.get('sportId')
+
+  const sportIdWhereClause = sportId ? { sportId: Number(sportId) } : {}
```

Comment:
i was like I don't love this variable name haha but what else do you call it lol